src/Cli/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommandParser.cs

```diff
@@ -61,6 +61,11 @@ internal static class ListPackageReferencesCommandParser
                 ArgumentHelpName = CommonLocalizableStrings.LevelArgumentName
             }.ForwardAsSingle(o => $"--verbosity:{o}");
 
+        public static readonly Option FormatOption = new ForwardedOption<string>("--format", LocalizableStrings.CmdFormatDescription)
+        {
+            ArgumentHelpName = LocalizableStrings.CmdConfig
+        }.ForwardAsMany(o => new[] { "--format", o });
```

Comment:
Should this be `ForwardAsSingle`? I'm not sure multiple format parameters are allowed, since they all write to stdout.

Reply:
You're right made it `ForwardAsSingle`.